packages/pwa/app/components/product-scroller/index.jsx

```diff
@@ -92,11 +91,7 @@ const ProductScroller = forwardRef(
                                             </Stack>
                                         </Stack>
                                     ) : (
-                                        <ProductTile
-                                            data-testid="product-scroller-item"
-                                            productSearchItem={product}
-                                            onClick={() => onProductClick(product)}
-                                        />
+                                        renderProduct(product)
```

Comment:
it's hard to customize the product tiles in the scroller so this is my attempt to decouple the two components...

feedback is welcomed